### PR TITLE
Implement darkmode on Learn/Platform Tools page 

### PR DESCRIPTION
**Pull request by oyenuga17**
@SaySayo @punchagan please help review

### CODE REVIEW

**Comment by sabine** [comment]
Hey @oyenuga17, there's a lot of conflicts here, I suppose you didn't want to include all those commits in the PR.

The only relevant one seems to be https://github.com/ocaml/ocaml.org/pull/1896/commits/1f2093c2e7594c25a4a3947e6b5f8321672f0f72.

The way to tidy this up:

make a copy of your branch, then go back to this PR branch. `git rebase -i main` (after updating `main` to be the same as ocaml/ocaml.org main branch) and remove all the unnecessary commits.

**Review by christinerose — APPROVED** [review]

File: src/ocamlorg_frontend/layouts/learn_layout.eml [hunk 1]
```diff
@@ -88,14 +88,14 @@ let sidebar
         <%s t %>
       </span>
     in
-    <a class="border-l-2 py-2 px-3 rounded-sm leading-6 <%s if current then {|font-bold text-primary bg-legacy-primary-100 border-primary|} else {|text-legacy-default hover:bg-gray-100 border-transparent|} %>" href="<%s href %>">
+    <a class="border-l-2 py-2 px-3 rounded-sm leading-6 font-normal <%s if current then {|text-title dark:text-dark-title bg-primary_nav_block_hover_10 dark:bg-primary_nav_block_hover_10 border-primary dark:border-dark-primary|} else {|text-content dark:text-dark-content hover:text-primary dark:hover:text-dark-primary border-transparent|} %>" href="<%s href %>">
```
Comment (SaySayo) [inline src/ocamlorg_frontend/layouts/learn_layout.eml:91]: It has the same background for light and dark mode.

File: src/ocamlorg_frontend/pages/home.eml [hunk 2]
```diff
@@ -353,7 +353,7 @@ Layout.render
     <div class="flex flex-col lg:flex-row justify-between lg:space-x-20 space-y-10 lg:space-y-0">
       <div class="shadow-lg p-10 bg-legacy-default dark:bg-legacy-dark-default rounded-xl lg:mb-0">
```
Comment (SaySayo) [inline src/ocamlorg_frontend/pages/home.eml]:
```suggestion
      <div class="shadow-lg p-10 bg-white dark:bg-dark-background rounded-xl lg:mb-0">
```

File: src/ocamlorg_frontend/pages/home.eml [hunk 3]
```diff
@@ -156,7 +156,7 @@ Layout.render
               </div>
               <div class="text-legacy-default font-bold">Powerful Type Safety Made Simple</div>
```
Comment (SaySayo) [inline src/ocamlorg_frontend/pages/home.eml]:
```suggestion
              <div class="text-content dark:text-dark-content font-bold">Powerful Type Safety Made Simple</div>
```

File: src/ocamlorg_frontend/pages/home.eml [hunk 4]
```diff
@@ -370,7 +370,7 @@ Layout.render
       </div>
       <div class="shadow-lg p-10 bg-legacy-default dark:bg-legacy-dark-default rounded-xl">
```
Comment (SaySayo) [inline src/ocamlorg_frontend/pages/home.eml]:
```suggestion
      <div class="shadow-lg p-10 bg-white dark:bg-dark-background rounded-xl">
```

File: src/ocamlorg_frontend/pages/learn.eml [hunk 5]
```diff
@@ -1,4 +1,4 @@
-let render
+ let render
```
Comment (SaySayo) [inline src/ocamlorg_frontend/pages/learn.eml]: Take out unnecessary white space

File: src/ocamlorg_frontend/pages/home.eml [hunk 6]
```diff
@@ -182,7 +182,7 @@ Layout.render
             <h3 class="text-legacy-default font-bold">
```
Comment (SaySayo) [inline src/ocamlorg_frontend/pages/home.eml]:
```suggestion
            <h3 class="text-content dark:text-dark-content font-bold">
```

**Review by SaySayo — CHANGES_REQUESTED** [review]

**Comment by SaySayo** [comment]
Check out the suggested changes @oyenuga17

File: src/ocamlorg_frontend/layouts/learn_layout.eml (hunk 1 above)
Comment (oyenuga17) [reply src/ocamlorg_frontend/layouts/learn_layout.eml:91]: Hi @SaySayo, they both have the same color according to the design

File: src/ocamlorg_frontend/pages/home.eml (hunk 2 above)
Comment (oyenuga17) [reply src/ocamlorg_frontend/pages/home.eml]: @SaySayo I think I added this changes by mistake. I'll discard the changes I made to this file for now and create a separate pr for this page when there's a design for it.

**Comment by oyenuga17** [comment]
closing this for now, will re-open when #1913 is merged

**Pull request closed without merging**